.github/workflows/aikido-scan.yml

```
@@ -0,0 +1,69 @@
+        with:
+          node-version: '22'
+
+      - name: Install Aikido CI API client
+        run: npm install --global @aikidosec/ci-api-client
+
+      - name: Run Aikido PR scan
+        if: ${{ github.event_name == 'pull_request' }}
+        env:
+          AIKIDO_CLIENT_API_KEY: ${{ secrets.AIKIDO_CLIENT_API_KEY }}
+          AIKIDO_REPOSITORY_ID: ${{ vars.AIKIDO_REPOSITORY_ID }}
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)